File: PRAK403_2510817210004_MuhammadRizkyramadhani.c
#include <stdio.h>

int main() {
    int a, b;

    printf("Selamat datang di program Angka Bersilangan!\n");
    printf("_________________________________________\n\n");

    printf("Masukkan angka pertama : ");
    scanf("%d", &a);
    printf("Masukkan angka kedua : ");
    scanf("%d", &b);

    printf("_________________________________________\n\n");

    printf("Berikut adalah hasilnya :\n");

    if (a < b) {
        for (int i = a; i <= b; i++) {
            printf("%d %d", i, a + b - i);
            if (i < b) {
                printf(" - ");
            }
        }
    } else {
        for (int i = a; i >= b; i--) {
            printf("%d %d", i, a + b - i);
            if (i > b) {
                printf(" - ");
            }
        }
    }

    return 0;
}

File: PRAK403_2510817210004_MuhammadRizkyramadhani.py
# Program Client Pak Samson
print('Selamat datang di program Angka Bersilangan!')
print('_________________________________________\n')
a = int(input("Masukkan angka pertama : "))
b = int(input("Masukkan angka kedua : "))
print('_________________________________________\n')

print('Berikut adalah hasilnya :')
if a < b:
    for i in range(a, b + 1):
        print(f"{i} {a + b - i}", end=' - ' if i < b else '')
else:
    for i in range(a, b - 1, -1):
        print(f"{i} {a + b - i}", end=' - ' if i > b else '')